.eslintrc.js

```diff
@@ -6,12 +63,32 @@ module.exports = {
         jest: true,
         "jest/globals": true
       }
+    },
+    {
+      files: ["*.ts", "*.tsx"],
+      parser: "@typescript-eslint/parser",
+      extends: [
+        ...commonExtends,
+        "plugin:@typescript-eslint/eslint-recommended",
+        "plugin:@typescript-eslint/recommended"
+      ],
+      plugins: [...commonPlugins, "@typescript-eslint"],
+      rules: {
+        ...commonRules,
+        "@typescript-eslint/ban-ts-comment": ["warn"],
+        "no-unused-vars": "off",
+        "@typescript-eslint/no-unused-vars": ["error", { argsIgnorePattern: "^_", varsIgnorePattern: "^_" }]
+      }
     }
   ],
   env: {
     browser: true,
     es2021: true
   },
+  globals: {
+    require: true,
+    module: true
```

Comment:
why do we need module as well?

Reply:
<img width="857" alt="image" src="https://user-images.githubusercontent.com/104433616/189516759-1d562ed3-33c0-4929-a594-f02fecc87ad4.png">
Without it I get this kind of errors

Reply:
we shouldn't use them in our code, so can you please ignore them in these specific files - (these files are for scripts)

Reply:
Done 👍